src/main/java/main/internet/shop/dao/jdbcimpl/ProductDaoJavaDataBaseConnectivityImpl.java

```diff
@@ -0,0 +1,113 @@
+package main.internet.shop.dao.jdbcimpl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import main.internet.shop.dao.ProductDao;
+import main.internet.shop.exception.DataProcessingException;
+import main.internet.shop.model.Product;
+import main.internet.shop.utils.ConnectionUtils;
+
+public class ProductDaoJavaDataBaseConnectivityImpl implements ProductDao {
+    @Override
+    public Product create(Product item) {
+        String productName = item.getName();
+        Double productPrice = item.getPrice();
+        String query = "INSERT INTO products (name, price) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtils.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query,
+                    Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, productName);
+            statement.setDouble(2, productPrice);
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
```

Comment:
in video it was while loop instead of if
I guess it should work this way also. 
need mentor explanation